picard/ui/options/interface.py

```diff
@@ -143,6 +143,8 @@ def fcmp(x):
         if not OS_SUPPORTS_THEMES:
             self.ui.ui_theme_container.hide()
 
+        self.ui.toolbar_multiselect.clicked.connect(self.multi_selection_warning)
```

Comment:
I would instead bind this to the `stateChanged`  signal. That way it does not only trigger on click but generally when the setting is toggled.

Reply:
I originally had it that way, but during testing if you enabled the option, then disabled it from the dialog, then enabled it again the dialog was not triggered.  I'll do some more testing to see if I can switch to using `stateChanged`.

Also, should it open the warning dialog when you open the options page (and it loads the saved values) if the option had been saved as enabled?

Reply:
> Also, should it open the warning dialog when you open the options page (and it loads the saved values) if the option had been saved as enabled?

I think not. So the signal must be connected after the initial value has been set.

Reply:
Now changed to use the `stateChanged` signal.  I also revised the logic of when the dialog is displayed to:

- Do not display warning when loading settings with option enabled
- Display warning when option state has changed and option state is
enabled
- Display warning when saving if option state is enabled and warning has
not been displayed this Options session

Reply:
> So the signal must be connected after the initial value has been set.

Okay, that makes much more sense than the way I had done it (using a `self.loading` variable).  I'll make the change.

Also, after thinking about it again, I'll remove the warning trigger when saving the settings.  The user will have received the warning at the point that they enabled the option initially, so shouldn't be bothered with it each time they save unrelated settings.

I'll also include some logic to not display the warning when changing profile settings (which I had forgotten about).

Reply:
I ended up connecting the signal in the initialization and then blocking it temporarily while loading values in order to avoid having the warning come up when enabling / disabling the option profiles.